about.html

```diff
@@ -0,0 +1,120 @@
+<!DOCTYPE html>
+<html lang="en">
+	<head>
+		<meta charset="UTF-8" />
+		<meta name="viewport" content="width=device-width, initial-scale=1.0" />
+		<link rel="stylesheet" href="./dist/index.css" />
+		<link rel="preconnect" href="https://fonts.googleapis.com" />
+		<link rel="preconnect" href="https://fonts.gstatic.com" crossorigin />
+		<link
+			href="https://fonts.googleapis.com/css2?family=Lato:ital,wght@0,100;0,300;0,400;0,700;0,900;1,100;1,300;1,400;1,700;1,900&family=Poppins:ital,wght@0,100;0,200;0,300;0,400;0,500;0,600;0,700;0,800;0,900;1,100;1,200;1,300;1,400;1,500;1,600;1,700;1,800;1,900&display=swap"
+			rel="stylesheet"
+		/>
+		<link
+			rel="stylesheet"
+			href="https://fonts.googleapis.com/css2?family=Material+Symbols+Outlined:opsz,wght,FILL,GRAD@20..48,100..700,0..1,-50..200"
+		/>
+		<title>About</title>
+	</head>
+	<body>
+		<header id="header">
+			<div class="top-banner">
+				<div class="banner">
+					<img src="./images/Facebook.svg" alt="" />
```

Comment:
- [ ] Looking at the video below, you can see that the `about.html` page has no CSS applied to its content, I'll suggest you look into this and style it according to the design. :+1: :+1: 

- - VIDEO REFERENCE:

https://github.com/sheezu900/module1-capstone-project/assets/107632780/e51bc196-aceb-48c3-a321-6f0671f3ca7e